modules/ipi-install-ipv6-network-requirements.adoc

```diff
@@ -0,0 +1,53 @@
+[discrete]
+== IPv6 considerations
+
+.SLAAC Addressing
+
+If you don't plan to use SLAAC footnote:[Stateless Address AutoConfiguration] addresses on your OpenShift node, then it should be disabled for `baremetal` networks, that means that if your network equipment is configured to send SLAAC addresses when replying to Route Advertisements that behavior should be changed, so it only sends the route and not the SLAAC address.
+
+You can install `ndptool` on your system in order to check what your RAs look like:
+
+[source,shell]
+----
+# Turn down/up baremetal iface on a master Node
+$ sudo nmcli con down "Wired connection 5" && sudo nmcli con up "Wired connection 5"
+Connection 'Wired connection 5' successfully deactivated (D-Bus active path: /org/freedesktop/NetworkManager/ActiveConnection/1983)
+Connection successfully activated (D-Bus active path: /org/freedesktop/NetworkManager/ActiveConnection/2044)
+
+# ndptool monitor on Helper node
+$ sudo ndptool monitor -t ra
+NDP payload len 80, from addr: fe80::c0a4:6464:bcb3:d657, iface: baremetal.153
+  Type: RA
+  Hop limit: 64
+  Managed address configuration: yes
+  Other configuration: no
+  Default router preference: medium
+  Router lifetime: 0s
+  Reachable time: unspecified
+  Retransmit time: unspecified
+  Source linkaddr: 1c:40:24:1b:0c:34
+  Prefix: 2620:52:0:1303::/64, valid_time: 86400s, preferred_time: 14400s, on_link: yes, autonomous_addr_conf: no, router_addr: no
+  Route: ::/0, lifetime: 0s, preference: low
+----
+
+The `ndptool monitor` should report `Managed address configuration: yes`.
+
+.Network Ranges and Configurations
+
+Different `baremetal` and `provisioning` networks are required for each environment, each environment will have a different IPv6 range for each one of those networks.
+
+In our configuration we used subinterfaces attached to two different physical interfaces, VLAN tagging was done at O.S. level (this required switch ports configured with `trunk` mode).
+
+Our different IPv6 networks were all routable but usually, the only routable networks are the `baremetal` ones.
```

Comment:
In Red Hat's test environment, different IPv6 networks were all routable, but usually the only routable networks are the `baremetal` networks.